pluct/schema.py

```diff
@@ -18,7 +18,7 @@ def __new__(cls, href, *args, **kwargs):
         if href in session.store:
             return session.store[href]
 
-        instance = super(Schema, cls).__new__(cls, href, *args, **kwargs)
+        instance = super(Schema, cls).__new__(cls)
```

Comment:
Yep, it's intentional. In python 3 if you override both `__new__` and `__init__` methods you should avoid pass extra arguments to them. Since that `__init__` method are called by `__call__` method(bound to the class) you don't need the same args at `__new__` and `__init__`.